### PR TITLE
Bug 1111597 - Set up Santali (sat) for production

### PR DESCRIPTION
**Pull request by flodolo**
Add to PROD_LANGUAGES and home page redirect rule.

### CODE REVIEW

File: etc/httpd/global.conf [hunk 1]
```diff
@@ -205,7 +205,7 @@ RewriteRule ^/(\w{2,3}(?:-\w{2})?/)?firefox/toolkit/download-to-your-devices(.*)
 # Has to be this way because of old
 # legacy platform url conflicts.
 # bug 921564
-RewriteRule ^/(ach|af|ak|an|ar|as|ast|az|be|bg|bn-BD|bn-IN|br|bs|ca|cs|csb|cy|da|de|dsb|el|en-GB|en-US|en-ZA|eo|es-AR|es-CL|es-ES|es-MX|et|eu|fa|ff|fi|fr|fy-NL|ga-IE|gd|gl|gu-IN|he|hi-IN|hr|hsb|hu|hy-AM|id|is|it|ja|ja-JP-mac|ka|kk|km|kn|ko|ku|lg|lij|lt|lv|mai|mk|ml|mn|mr|ms|my|nb-NO|nl|nn-NO|nso|oc|or|pa-IN|pl|pt-BR|pt-PT|rm|ro|ru|sah|si|sk|sl|son|sq|sr|sv-SE|sw|ta|ta-LK|te|th|tr|uk|ur|uz|vi|wo|xh|zh-CN|zh-TW|zu)(/?)$ /b/$1$2 [PT]
+RewriteRule ^/(ach|af|ak|an|ar|as|ast|az|be|bg|bn-BD|bn-IN|br|bs|ca|cs|csb|cy|da|de|dsb|el|en-GB|en-US|en-ZA|eo|es-AR|es-CL|es-ES|es-MX|et|eu|fa|ff|fi|fr|fy-NL|ga-IE|gd|gl|gu-IN|he|hi-IN|hr|hsb|hu|hy-AM|id|is|it|ja|ja-JP-mac|ka|kk|km|kn|ko|ku|lg|lij|lt|lv|mai|mk|ml|mn|mr|ms|my|nb-NO|nl|nn-NO|nso|oc|or|pa-IN|pl|pt-BR|pt-PT|rm|ro|ru|sah|sat|si|sk|sl|son|sq|sr|sv-SE|sw|ta|ta-LK|te|th|tr|uk|ur|uz|vi|wo|xh|zh-CN|zh-TW|zu)(/?)$ /b/$1$2 [PT]
```
Comment (pmclanahan) [inline etc/httpd/global.conf]: Are there still home page locales that shouldn't get to the bedrock one?

**Comment by flodolo** [comment]
That's a good question, I don't think so. 
My understanding, reading the comments, is that this was needed somehow.

@pascalchevrel Thoughts?

**Comment by pascalchevrel** [comment]
There are no locales that need the old PHP homepage, so that rewrite should be for all locales now.

**Comment by flodolo** [comment]
I removed the change to the redirect rule, for clarity I'll file a different bug to apply the rule to all locales.

**Comment by pmclanahan** [comment]
We've decided not to do the wild-card locale home page rule change for now. Please change that rule in this PR if you still need to.

**Comment by flodolo** [comment]
Restored change to rewrite rule.

**Comment by pmclanahan** [comment]
Merged as d627564d352f0f0af8a5e06e768c4c7f6035fbf4. Thanks.

**Pull request closed without merging**